drivers/SmartThings/zigbee-motion-sensor/src/aqara/motion-illuminance/init.lua

```diff
@@ -56,20 +62,18 @@ local function motion_illuminance_attr_handler(driver, device, value, zb_rx)
   -- The low 16 bits for Illuminance
   -- The high 16 bits for Motion Detection
 
-  if value.value > MOTION_DETECTED_UINT32 then
```

Comment:
I think you probably should just change this to `>=` to avoid the possibility that you emit a negative value for lux.

Reply:
You are right. 
I also changed to the default value of detection frequency interval variable to the device's default value (1 min).

Reply:
Hi @sbkimaqara, could you update like below?
```diff
-  if value.value >= MOTION_DETECTED_UINT32 then
+  if value.value >= MOTION_DETECTED_UINT32 then
```

Reply:
The IF conditional statement is not needed. The handler is called when motion is detected, and values include motion and illuminance values. So the motion UI is updated without conditional statements and the illuminance UI updated using low bit of received value.

Reply:
> You are right. I also changed to the default value of detection frequency interval variable to the device's default value (1 min).

Sorry. I think my answer makes misunderstood you. Please refer to the comments below.

Reply:
I'll push again after cleaning the code.